js/stores/simulation-store.ts

```diff
@@ -198,6 +199,9 @@ export class SimulationStore {
   @action.bound setHotSpot(data: { position: THREE.Vector3; force: THREE.Vector3 }) {
     this.currentHotSpot = null;
     workerController.postMessageToModel({ type: "setHotSpot", props: data });
+    // TODO: better synchronization approach
+    // wait for model to update hotSpot before triggering updates
+    setTimeout(() => ++this.hotSpotChanges, 100);
   }
```

Comment:
Right, I'd hope that the whole `hotSpotChanges` thing could be done somehow differently. Maybe hot spots could just become observable properties of plate store. But this can be refactored (if possible at all) in the subsequent PRs.